Project.toml

```diff
@@ -10,10 +10,10 @@ Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-Reexport = "0.2"
-Requires = "0.5.2"
 ChainRulesCore = "0.4"
 FiniteDifferences = "^0.7"
+Reexport = "0.2"
+Requires = "0.5.2"
```

Comment:
Prob allow 0.5.2 or 1

Reply:
Is there a reason why we're locking to `0.5.2` specifically?